packages/ui/src/components/NavMenu/index.tsx

```diff
@@ -20,17 +20,16 @@ export const NavMenu = forwardRef<HTMLDivElement, NavMenuProps>(
   }
 )
 
-export const NavMenuItem = forwardRef(
-  ({
-    children,
-    className,
-    active,
-    ...props
-  }: PropsWithChildren<{
+interface NavMenuItemProps
+  extends PropsWithChildren<{
     className?: string
     active: boolean
-  }>) => (
+  }> {}
+
+export const NavMenuItem = forwardRef<HTMLLIElement, NavMenuItemProps>(
+  ({ children, className, active, ...props }, ref) => (
     <li
+      ref={ref}
```

Comment:
is this part of the playwright stuffs too?

Reply:
It logged an console error, so I fixed it to remove it. Could go in a separate PR also.